components/dialog/test/dialog.test.js

```diff
@@ -14,7 +14,7 @@ describe('d2l-dialog', () => {
 
 	});
 
-	describe.skip('focus management', () => {
+	describe('focus management', () => {
```

Comment:
I'm not sure why this was disabled. I enabled it and all tests pass.

Reply:
Ah everything is failing on webkit. I see. I will disable again.

Reply:
Oh I see: https://github.com/BrightspaceUI/core/pull/1351#issuecomment-843579774

Reply:
Yeah not sure... looks like they've been skipped since @emil-furniss created [in this change](https://github.com/BrightspaceUI/core/commit/cd63c0567faeb364f19cf47e71111537e6b53be6#diff-12b7c6cce35edaf42462efa513d894e23b91ac2d9277617684b570b52a31669d).